align.go

```diff
@@ -69,7 +69,7 @@ func alignTextVertical(str string, pos Position, height int, _ *termenv.Style) s
 	case Top:
 		return str + strings.Repeat("\n", height-strHeight)
 	case Center:
-		var topPadding, bottomPadding = (height - strHeight) / 2, (height - strHeight) / 2
+		topPadding, bottomPadding := (height-strHeight)/2, (height-strHeight)/2
```

Comment:
Looks good, @mikelorant, however can you add a nolint directive for the magic number `2` here?